app-admin/systemrescuecd-x86/systemrescuecd-x86-4.6.0.ebuild

```diff
@@ -29,6 +35,8 @@ pkg_postinst() {
 	if ! has_version ">${CATEGORY}/${PF}"; then
 		ln -f -s -v "${P}.iso" "${f}" || die
 	fi
+
+	elog "You can set custom boot options in /etc/default/systemrescuecd"
```

Comment:
Since this applies to grub users only, you should say so. Also probably doesn't make much sense to print it when grub's not installed.